bgpd/rfapi/vnc_zebra.c

```diff
@@ -872,7 +872,7 @@ static zclient_handler *const vnc_handlers[] = {
 void vnc_zebra_init(struct event_loop *master)
 {
 	/* Set default values. */
-	zclient_vnc = zclient_new(master, &zclient_options_default,
+	zclient_vnc = zclient_new(master, &zclient_options_auxiliary,
```

Comment:
Where is the zapi communication of this change into zebra and it respecting this?

Reply:
Why would this have any effect on zebra? The goal is to not invoke the common `lib/` callbacks on the receiving end in the daemon. There can still be other code that wants to process the same notifications (e.g. running multithreaded), just without duplicating/going into `lib/` structures.